jenkinsbuilder/weekly-scan.py

```diff
@@ -8,7 +8,7 @@
 """
 
 import beanstalkc
-import container_pipeline.lib.dj
+# import container_pipeline.lib.dj
```

Comment:
commenting this out won't configure the django for weekly scan, please uncomment this line to load all the modules, check the dj script
```
$ cat container_pipeline/lib/dj.py 
import django
import os


def load():
    os.environ.setdefault('DJANGO_SETTINGS_MODULE',
                          'container_pipeline.lib.settings')
    django.setup()

load()
```

Reply:
we are importing this but not calling the load function anywhere. ok uncommenting it.

Reply:
@bamachrn 

> we are importing this but not calling the load function anywhere.

I don't think you will need to call the `load()` function explicitly. You can see, the load function is explicitly called in `dj.py`, so even if you import the dj script, the `load` will be called at the start of the file.